docs/rest_api.md

```diff
@@ -533,7 +533,27 @@ Response body:
 		   "networks": {"vlan/native": "pxe", "vlan/235": "storage"}},
                   {"label": "nic2", 
 		   "macaddr": "12:34:56:78:90", 
-		   "networks":{"vlan/native": "public"}}],
+		   "networks":{}}],
+	 "metadata":{"EK":"pk"}
+	}
+
+
+Response body when run by an admin:
+
+There are two additional fields, `port` and `switch`.
+
+    {"name": "node1",
+	 "project": "project1",
```

Comment:
Indentation is rather wonky. I could spend time picking it apart, but if you just pipe it through `python -m json.tool` it will output something reasonable. Same goes for the other one.